script.js

```diff
@@ -1,6 +1,27 @@
 /* eslint-disable linebreak-style */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-loop-func */
+
 const hamburger = document.querySelector('.hamburger');
 const toolbar = document.querySelector('.mobile-menu');
+const form = document.getElementById('contact-form');
+const errorMsg = document.getElementById('text');
+const inputName = document.getElementById('name');
+const inputEmail = document.getElementById('email');
+const inputMsg = document.getElementById('msg');
+
+const projectCard = [
+  {
```

Comment:
Pop-up for mobile also does not follow the design. The heading and technology list should be in the middle of the page. The page becomes completely blank after clicking on `X` to close the window. Moreover heading and the list should not show twice as it can be seen in the project.  KIndly make it look exact same as in the design
![mobile1](https://user-images.githubusercontent.com/35504149/136610687-8e57c333-0756-4386-a925-abf381501053.png)
![mobile2](https://user-images.githubusercontent.com/35504149/136610699-8de5872a-d5c2-42a0-a906-5d44ddba386f.png)
.
![blank](https://user-images.githubusercontent.com/35504149/136610710-cd659704-6161-4554-b929-6c32421f0757.png)